wagtail_wordpress_import/prefilters/linebreaks_wp_filter.py

```diff
@@ -81,7 +81,7 @@ def filter_linebreaks_wp(pee, options=None):
         pee,
     )
     pee = re.sub(
-        r"(</?" + allblocks + "[^>]*>)\s*</p>",
+        r"(</?" + allblocks + "[^>]*>)\s*</p>",  # noqa
```

Comment:
**Suggestion**: You can satisfy Flake8 by making both the strings raw:

```suggestion
        r"(</?" + allblocks + r"[^>]*>)\s*</p>",
```

Reply:
Thanks Nick: Fixed